doc/index.md

```diff
@@ -113,4 +113,41 @@ public class Connection
         var myResult = await rpc.InvokeWithParameterObjectAsync<string>("test/InvokeTestMethod");
     }
 }
+```
+
+## Crashing the process on exception
+In some cases, you may want to immediately crash the server and client processes if certain exceptions are thrown. In this case, overriding the `IsFatalException` method will give you the desired functionality. `IsFatalException` provides an exception filter through which you can access and respond to exceptions as they are caught.
```

Comment:
We may want to remove the "exception filter" reference since it is not always called from an exception filter.